ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,15 +14,19 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
+<form {{on "submit" (perform this.setAssociation)}}>
   <MessageError @errorMessage={{this.error}} />
 
   {{#if this.syncedSecret}}
-    <Hds::Alert @type="inline" @color="success" as |A|>
-      <A.Title>Successfully synced a secret</A.Title>
+    <Hds::Alert @type="inline" @color="success" @icon="sync" as |A|>
+      <A.Title>Sync initiated</A.Title>
       <A.Description data-test-sync-success-message>
-        Sync operation successfully initiated for "{{this.syncedSecret}}". You can continue on this page to sync more
-        secrets.
+        Sync operation successfully initiated for
+        <Hds::Link::Inline
+          @isRouteExternal={{true}}
+          @route="kvSecretDetails"
+          @models={{array this.mountPath this.syncedSecret}}
+        >{{this.syncedSecret}}</Hds::Link::Inline>. You can continue on this page to sync more secrets.
```

Comment:
Gif in PR description is outdated, there is no trailing space anymore:
<img width="1055" alt="Screenshot 2023-12-28 at 1 51 32 PM" src="https://github.com/hashicorp/vault/assets/68122737/967e3be4-43ee-425f-ac48-16fc2e239c10">